applications/GeoMechanicsApplication/tests/test_element_lab/oedometer_ULFEM/README.md

```diff
@@ -0,0 +1,30 @@
+# Oedometer ULFEM test
+
+This test simulates an oedometer lab test. The UPwUpdatedLagrangianElement2D3N element is used to simulate the soil
+sample in a very simple 4-noded mesh, as displayed in the following figure:
+
+![MeshStructure](MeshStructure.svg)
+
+## Setup
+
+The test is performed in a single stage, with the following conditions:
+
+-   Constraints:
+    -   The X and Y displacement in the bottom two nodes (3 and 4) are fixed.
+    -   The X displacement in all nodes is fixed.
+-   Material:
+    -   The material is described using a linear elastic material with a GeoLinearElasticPlaneStrain2DLaw.
```

Comment:
Can you also add the Young's Modulus here, E, here, as this will enable users to assert the displacement below. Due to being Linear Elastic.

Reply:
This indeed clarifies a lot, added, thanks for the suggestion!